src/gpuarray_buffer_cuda.c

```diff
@@ -147,13 +148,49 @@ static int setup_lib(void) {
         return error_set(global_err, GA_IMPL_ERROR, "cuDriverGetVersion failed");
       major = tmp / 1000;
       minor = (tmp / 10) % 10;
+      #if defined(WIN32) || defined(_WIN32) || defined(WIN64) || defined(_WIN64) || defined(__APPLE__)
```

Comment:
It is true that we only look at a specific file for windows, but I would rather not spread the assumption here.

We may devise a way to load a specific version of cuda on linux/mac later, and I would rather that the necessary components are in place at the upper levels.  Also this makes the code more complex for no benefit.